plugins/inputs/intel_rdt/README.md

```diff
@@ -46,29 +53,29 @@ More about Intel RDT: https://www.intel.com/content/www/us/en/architecture-and-t
 ### Configuration
 ```toml
 # Read Intel RDT metrics
-[[inputs.IntelRDT]]
-	## Optionally set sampling interval to Nx100ms. 
-	## This value is propagated to pqos tool. Interval format is defined by pqos itself.
-	## If not provided or provided 0, will be set to 10 = 10x100ms = 1s.
-	# sampling_interval = "10"
+[[inputs.intel_rdt]]
```

Comment:
@p-zak can you clarify the name of this plugin? Will need to be updated in the `SampleConfig()` section of `intel_rdt.go`

Reply:
I will check that.

Reply:
fyi this plugin name line doesn't go in SampleConfig.

Reply:
Yes, it should be `intel_rdt`. I think it should also be changed in https://github.com/influxdata/telegraf/blob/master/plugins/inputs/intel_rdt/intel_rdt.go#L544 and we are all set :)